src/message.js

```diff
@@ -59,8 +59,8 @@ Globalize.loadMessages = function( json ) {
  * Format a message given its path.
  */
 Globalize.messageFormatter =
-Globalize.prototype.messageFormatter = function( path ) {
-	var cldr, formatter, message, pluralGenerator, returnFn,
+Globalize.prototype.messageFormatter = function( path, isStructuredBidiText ) {
```

Comment:
Instead of the `isStructuredBidiText` argument, let's use an `options` object like `{explicitBiDi: true}`. Feel free to suggest a better (more appropriate) option name.